test/specs/shallow-mount.spec.js

```diff
@@ -130,7 +136,11 @@ describeRunIf(process.env.TEST_ENV !== 'node', 'shallowMount', () => {
     expect(wrapper.find({ name: 'Foo' }).exists()).to.equal(true)
     expect(wrapper.find('.new-example').exists()).to.equal(true)
     expect(wrapper.html()).to.equal(
-      '<foo-stub>\n' + '  <p class="new-example">text</p>\n' + '</foo-stub>'
+      '<foo-stub>\n' +
+        '  <template-stub slot="newSyntax">\n' +
```

Comment:
The need to update all the assertions with `template-stub` worry me. This will be a breaking change :/